### ISSUE TITLE
UDButton onclick - script block command stopped working after build of May-25

### ISSUE THREAD

**Issue by wsl2001** [ironmansoftware/universal-dashboard#832]
I have deployed latest fix as of today and it seems like there is issues with show-udtoast title not working if passed string object to it, only works if its blank or a text typed in it 
e.g   Show-UDToast -Message "Service Started!" -MessageColor Green -Title $_.Name -Position topCenter -Duration 2500    does not work

![image](https://user-images.githubusercontent.com/47834749/58377281-bb686800-7f4a-11e9-86c0-6bf9b99386da.png)

Show-UDToast -Message "Service Started!" -MessageColor Green -Title "Test" -Position topCenter -Duration 2500   works .

also button onclick to execute a command like 

Get-Service -ComputerName "server1" -Name $_.Name | Start-Service   does not work anymore.

tried using debug could not get any information.

**Comment by adamdriscoll** [comment]
I can't reproduce this. can you please try a nightly build? https://forums.universaldashboard.io/t/nightly-releases/756

**Comment by adamdriscoll** [comment]
Can you post more of your script? What is in the $_ variable?

**Comment by adamdriscoll** [comment]
I can reproduce this if I pass $null to title.

```
Show-UDToast -Message "test" -Title $null
```

It's likely that the $_ variable is null for some reason.

**Comment by adamdriscoll** [comment]
I actually have a theory why this could happen and PR #836 might fix it. Might want to try tonight's nightly build.

**Comment by wsl2001** [comment]
here i an example that still does not work. it was working before on all previews versions up to 2.4 beta one.
now the variable $_.Name wont pass to onclick script block

$Schedule3 = New-UDEndpointSchedule -Every 10 -Second


$Services = New-UDEndpoint -Schedule $Schedule3 -Endpoint {


              
             $Cache:Services = (get-Service -ComputerName "server1") | Select-Object Name,Status | Sort-Object Name | % { 

                    
                     
                    
                    [PSCustomObject]@{
                        TaskName = $_.Name
                        State = $_.Status
                        Select = if($_.Status -eq 'Stopped'){
                        New-UDButton -Text "Start" -OnClick {
                        
                        Get-Service -ComputerName "server1" -Name $_.Name | Start-Service

                        Show-UDToast -Message "Task Started!" -MessageColor Green -Title $_.Name -Position topCenter -Duration 2500
                       
                          }
                        } else{$null}     
                      }      
                    }
                }
            


$Page3 = New-UDPage -Name "Services" -Icon anchor -Content {

         New-UDElement -Id "Services" -Tag div2 -EndPoint { 

         New-UDGrid -Id "Grid3" -Title "All Services" -Headers @("Name","Status","Select") -Properties @("Name","Status","Select") -Endpoint {
        
             $Cache:Services | Out-UDGridData 
                 
                        
                        
          } 
       }

       New-UDButton -Text "Refresh" -OnClick {
            Sync-UDElement -Id "Grid3" -Broadcast
       }
    }

Start-UDDashboard -Wait -Endpoint @($Services) -Content{  
New-UDDashboard -Pages @($Page3) -Title "Tasks" -Color '#FF050F7F'
} -Port 80 -Name Tasks

**Comment by wsl2001** [comment]
so far today release seems to be fine.

**Comment by wsl2001** [comment]
works on server 2012 but dashboard wont load on server 2019 IIS

![image](https://user-images.githubusercontent.com/47834749/58396762-069e7b80-801c-11e9-9ec8-4f143e7868f6.png)

**Comment by adamdriscoll** [comment]
Ok. Thanks for letting me know. I will take a look. Glad the button is working at least.

**Comment by adamdriscoll** [comment]
I just tried on 2019 IIS and it's working for me. Can you please grab a log from UD? It should have the error that is actually happening. You can use Enable-UDLogging in the dashboard.ps1 file to output to a file. 

I take it you're just using the script above?

**Comment by adamdriscoll** [comment]
I managed to get your script running in my 2019 IIS lab as well. I had to make a couple tweaks. I had to customize the application pool user so that the user had permissions to start services. 

```
$Schedule3 = New-UDEndpointSchedule -Every 10 -Second

$Services = New-UDEndpoint -Schedule $Schedule3 -Endpoint {

         $Cache:Services = (get-Service) | Select-Object Name,Status | Sort-Object Name | % { 
                [PSCustomObject]@{
                    TaskName = $_.Name
                    State = $_.Status.ToString()
                    Select = if($_.Status.ToString() -eq 'Stopped'){
                        New-UDButton -Text "Start" -OnClick (
                            New-UDEndpoint -Endpoint {
                                Get-Service -Name $ArgumentList[0] | Start-Service
                                Show-UDToast -Message "Task Started!" -MessageColor Green -Title $ArgumentList[0] -Position topCenter -Duration 2500
                                Sync-UDElement -Id "Grid3" -Broadcast
                            } -ArgumentList $_.Name
                        )
                    } else{$null}     
                  }      
                }
            }
$Page3 = New-UDPage -Name "Services" -Icon anchor -Content {

     New-UDElement -Id "Services" -Tag div2 -EndPoint { 

     New-UDGrid -Id "Grid3" -Title "All Services" -Headers @("Name","Status","Select") -Properties @("TaskName","State","Select") -Endpoint {
    
         $Cache:Services | Out-UDGridData 
             
                    
                    
      } 
   }

   New-UDButton -Text "Refresh" -OnClick {
        Sync-UDElement -Id "Grid3" -Broadcast
   }
}
Start-UDDashboard -Wait -Endpoint @($Services) -Content{
New-UDDashboard -Pages @($Page3) -Title "Tasks" -Color '#FF050F7F'
} -Port 80 -Name Tasks
```

**Comment by wsl2001** [comment]
issue resolved in 526.3 nightly build. Thanks
